src/vcpkg/paragraphs.cpp

```diff
@@ -536,9 +536,9 @@ namespace vcpkg::Paragraphs
             {
                 for (auto&& error : results.errors)
                 {
-                    print2(Color::warning, "Warning: an error occurred while parsing '", error->name, "'\n");
+                    msg::println_warning(msgErrorWhileParsing, msg::path = error->name);
                 }
-                print2(Color::warning, "Use '--debug' to get more information about the parse failures.\n\n");
+                msg::println_warning(msgGetParseFailureInfo);
```

Comment:
You're missing one `\n` here.

Reply:
It's ok. I decided to remove it. We aren't consistent about when we add newlines/ how many newlines, and it didn't seem necessary to have two newlines.